inc/util/file_utils.hpp

```diff
@@ -37,6 +43,25 @@ namespace ebi
         }
     }
 
+    inline void open_local(std::ifstream & input,
```

Comment:
I don't see the point of this function, why not use open_file directly? if you want to highlight it's a local file, you can rename open_file to open_local_file. If your concern is const-correctness, you can add const to the old function